hack/gosec.sh

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env bash
 
 source hack/common.sh
-export ARTIFACTS=${ARTIFACTS:-$KUBEVIRT_DIR/_out/artifacts}
 
 mkdir -p $ARTIFACTS
 
 echo "Run go sec in pkg"
 cd $KUBEVIRT_DIR/pkg
 
-gosec -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
+gosec -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
```

Comment:
@xpivarc Just a small one - can you please add a newline at the end?

Reply:
done